docs/cloud/managing-airbyte-cloud.md

```diff
@@ -276,6 +277,38 @@ To refresh the source schema:
 
 2. If there are changes to the schema, the **Refreshed source schema** dialog displays them.
 
+## Enable schema update notifications
+Get schema update notifications when your source schema changes.
+
+To get notified when your schema changes: 
+1.  Make sure you have [webhook notifications](https://docs.airbyte.com/cloud/managing-airbyte-cloud#manage-airbyte-cloud-notifications) set up.
+
+2. On the [Airbyte Cloud](http://cloud.airbyte.io/) dashboard, click **Connections** and click the connection you want to change.
+
+3. Click the **Settings** tab on the Connection page.
+
+4. Toggle **Schema update notifications**.
+
+## Review schema changes
+Airbyte detects changes in your source schema and allows you to review the changes. 
+
+| Breaking changes                                                                                                                                                      | Non-breaking changes                                                                                                                                                                                                                  |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| If your connection’s source schema has breaking changes, the syncs will be stopped and you must review the changes before continuing syncs or editing the connection. | If you choose to ignore non-breaking changes in your connection's source schema, the syncs will continue.  You can change how Airbyte handles syncs when it detects non-breaking changes in the source schema by [editing the stream configuration](#edit-stream-configuration). |
+
+To review schema changes: 
```

Comment:
@edmundito @alovew Are the steps for reviewing a non-breaking change and a breaking change the same? I couldn't find a breaking change in a workspace to test this.

Reply:
If there are breaking schema changes, the user will have to fix what is causing the connection to be broken. This will mean either selecting a new cursor field or new primary key.